### DISCUSSION TITLE
how to keep it contained in that one file

### DISCUSSION THREAD

**Discussion by lnxfsf** [alefragnani/vscode-bookmarks#736] [Q&A] [answered]
When I click ctrl + j, ctrl + l , it goes to bookmarks that are outside of current file. 
I don't want that. I want it to go through all bookmarks but only in current file. 

How to do it ?

**Comment by alefragnani** [comment, marked as answer]
Hi @lnxfsf ,

Simply change the `bookmarks.navigateThroughAllFiles` setting to `false`. You can combine with `bookmarks.wrapNavigation` to wrap around at the first and last bookmarks.

Hope this helps
